### DISCUSSION TITLE
到底是谁！！

### DISCUSSION THREAD

**Discussion by 76asurname76** [share121/inter-knot#5804] [公告]
![IMG_20241222_165648](https://github.com/user-attachments/assets/e574b9ec-4511-439f-9157-5bdf7389f8fc)
到底是谁在把风滚草运到电梯上？别让我逮到你，逮到你指定让霸主去你家门口！

**Comment by FurinaOne** [comment]
算盘打的我在六分街都听见了

**Comment by fa-e-tong-LKN** [comment]
1 vote()

**Comment by FeiLiFL** [comment]
好像是一个蓝头发的小妹妹踢的，就是上个月和她哥开着辆商务车来外环的那个，大半夜12点也不睡觉搁那和卡吕冬之子那帮子在街上瞎逛，听别人说好像还是个麦片的？（存疑）

**Reply by feather000A** [reply]
典中典，麦片兄妹
